magento/magento2
# Multishipping Checkout Store id is saved NULL in sales_order_item Table: FIXED

During Multishipping Checkout Store Id is saving NULL in **sales_order_item** Table. Fixed in this PR.



### Description (*)

I was fixing the Quote item id issue in this PR #19192 , found there is also Store id issue with Multishipping checkout. There was no **store_id** column in **quote_address_item**  that's why
store_id values saved NULL. I have added **store_id** column in **quote_address_item** table in this PR.
### Fixed Issues (if relevant)

1. 
2. 

### Manual testing scenarios (*)

1. After updating code i can check store_id saved correctly with **Multishipping Checkout**  https://prnt.sc/llfjzv 
2. ...

### Contribution checklist (*)
